hack/perf-ibmcloud-openshift.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+
+##############################################################################
+# perf-ibmcloud-openshift.sh
+#
+# Setup a performance testing environment on Openshift in IBM Cloud.
+#
+# Pass --help for usage information.
+#
+##############################################################################
+
+set -eu
+
+errormsg() {
+  echo -e "\U0001F6A8 ERROR: ${1}"
+  exit 1
+}
+
+infomsg() {
+  echo -e "\U0001F4C4 ${1}"
+}
+
+SCRIPT_DIR=$( dirname -- "$0"; )
+
+DEFAULT_HELM_CHARTS_REPO="${SCRIPT_DIR}/../helm-charts"
```

Comment:
I think this default is incorrect. Assuming one has a kiali dev environment where you clone all the kiali repos in peer directories, such as:

/source/kiali
/source/kiali-operator
/source/helm-charts

Then this should be:

```suggestion
DEFAULT_HELM_CHARTS_REPO="${SCRIPT_DIR}/../../helm-charts"
```

The SCRIPT_DIR will be `/source/kiali/hack`, thus you need to go up TWO directories (`../..`) in order to get to the helm-chart repo.